hack/tool/validation/tf_validation_backend/README.md

```diff
@@ -0,0 +1,24 @@
+# tf_validation_backend
```

Comment:
What does `validation` mean?

Reply:
`validation` means that if the backend block is invalid, the tool will report an error.
emmm,  maybe it woudl be better to use `fetch`?